src/index.ts

```diff
@@ -8,6 +8,13 @@ declare module 'pinia' {
      */
     persist?: boolean | PersistedStateOptions
   }
+  export interface PiniaCustomProperties {
+    /**
+     * Restore state manually.
+     * @docs https://github.com/prazdevs/pinia-plugin-persistedstate.
```

Comment:
no need for the docs link.  but this doc doesnt feel informative.
maybe smth like "Forces the pinia store to be rehydrated with stored data"  and i'd like to have some warning that this shouldnt be used in most cases

Reply:
Ok, the doc is changed